pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -247,7 +247,7 @@ public static Timestamp toTimestamp(long millis) {
   /**
    * Converts Timestamp to epoch millis
    */
-  @ScalarFunction
```

Comment:
Why are they annotated as nullable? The input should not be null

Reply:
@Jackie-Jiang `fromTimestamp` takes Timestamp (object) that can be null but cannot handle null (returns primitive long). Btw, I see `fromTimestamp` is not being used so I removed the added annotation.

I applied the same logic to other functions: `ago`, `timezoneHour`, `timezoneMinute` and `dateTimeConvert`.